modules/deleting-a-connection.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+
+[id="deleting-a-connection_{context}"]
+= Deleting a connection
+
+[role='_abstract']
+You can delete connections from your data science projects to help you remove connections that are no longer relevant to your work.
```

Comment:
Suggestion: _You can delete connections that are no longer relevant to you work from your data science projects_